README.md

```diff
@@ -149,4 +149,4 @@ try serializer.serialize(users)
 
 ## License
 
-JSONAPISerializer is available under the MIT license. See the LICENSE file for more information.
\ No newline at end of file
+JSONAPISerializer is available under the MIT license. See the LICENSE file for more information.
```

Comment:
Guessing GitHub cleaned up a newline at the end of the file here, but the diff isn't showing it. I didn't edit this line.